### DISCUSSION TITLE
Returns null when query for the 2nd time in a 1-1 relationship

### DISCUSSION THREAD

**Discussion by ThemeMonkey-Dev** [prisma/prisma#8749] [Q&A]
There are products in the products table.
There are orders in the order_product table.

With the barcode of the product belonging to the order in the order_product table, I access the brand of the product from the products table with the 1-1 model.

I also make the query to the order_product table with the ones whose brands are "x" in the products table.

However,
For example, when I query the product with 1234 barcodes in the order, the first time it returns, the data in table a comes. but if 1234 barcode is queried again it comes back as null.

If the product in the products table has not been queried before, the data is coming.

Look, the product barcodes on the printout are the same.

Output:
```
[
...,
{
  id: 1032,
  order_id: '56123',
  product_code: '86993',
  barcode: '86993', 
  product_name: 'Nelly Blue Hair Care Serum 3x33 ml (buy 3, share for 2) - Blue Water'',
  price: 127.9,
  quantity: 1,
  tax: 18,
  product: null
},
{
  id: 994,
  order_id: '1234',
  product_code: '86993',
  barcode: '86993',
  product_name: 'Nelly Blue Hair Care Serum 3x33 ml (buy 3, share for 2) - Blue Water'',
  price: 127.9,
  quantity: 1,
  tax: 18,
  product: {
    barcode: '86993',
    name: 'Nelly Blue Hair Care Serum 3x33 ml (buy 3, share for 2) - Blue Water',
    Stock: 30,
    Brand: 'Nelly'
  }
}
]
```

Schema:
```prisma
model order_product {
..
product products?
}
```

model file
```ts
const getOrdersByBrand = async(brands) => {

  const ordersByBrand = await prisma.order_product.findMany({
        where: {
            product: {
                Brand: { in: brands
                }
            }
        },
        include: {
            product: true
        },
    })

    return { ordersByBrand }
}
```

**Comment by ryands17** [comment]
@ThemeMonkey-Dev 👋 

I would need your products schema as well to test this out.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
